README.md

```diff
@@ -152,11 +152,11 @@ The shared storage worklet invocation methods (`addModule`, `createWorklet`, and
     *   Options include:
         *   `withLock`: acquire a lock on the designated resource before executing. See [Locking for modifier methods](#locking-for-modifier-methods) for details.
 *   `window.sharedStorage.batchUpdate(methods, options)`
-    *   Execute `methods` in order.
-    *   `methods` is an array of method objects defining the operations to perform. Each object must be one of the following types: `SharedStorageSetMethod`, `SharedStorageAppendMethod`, `SharedStorageDeleteMethod`, or `SharedStorageClearMethod`. Each method object's constructor accepts the same parameters as the corresponding individual method (e.g., `set`, `append`, `delete`, `clear`).
+    *   Execute `methods` in order. All updates within a `batchUpdate()` call are executed as a single unit of work.
```

Comment:
Suggest appending "single unit of work. If any method fails, none of them will commit."

Reply:
Done.